src/components/form/form-test.stories.tsx

```diff
@@ -381,15 +381,20 @@ DefaultWithPager.storyName = "default with pager";
 export const MockFormForAriaLiveDemo = () => {
   const [textareaValue, setTextareaValue] = useState("");
   const [textboxValue, setTextboxValue] = useState("");
-  const [textEditorValue, setTextEditorValue] = useState(
-    EditorState.createEmpty(),
-  );
+  const [textEditorValue, setTextEditorValue] = useState(() => {
+    // 'empty' editor
+    const value =
+      '{"root":{"children":[{"children":[],"direction":null,"format":"","indent":0,"type":"paragraph","version":1}],"direction":null,"format":"","indent":0,"type":"root","version":1}}';
```

Comment:
suggestion: it might be worth using the createEmpty util you've added in the docs as well